shared_workflow/shared_template.py

```diff
@@ -83,8 +83,12 @@ def generate_command(
     command_parts.append(command_template.format(**template_parameters))
 
     for key in add_args:
+        if (
+            add_args[key] is False
+        ):  # Don't add store_true type arg at all if False.(This repo has no use of store_false)
+            continue
```

Comment:
Good idea. It sounds like we need to give default None to all store_true type arguments that will rely on write_sl_script()?

Reply:
Actually, I think this will be fine. If we do end up needing to pass False as a value in future we can look at changing our keywords then.
No changes needed here